build/yaml/samples/nsx_v1alpha1_vpcnetworkconfigurations.yaml

```diff
@@ -21,5 +17,6 @@ metadata:
   name: vpc-network-config-with-pre-created-vpc
 spec:
   vpc: /orgs/default/projects/proj-1/vpcs/vpc-1
-  defaultIPv4SubnetSize: 32
-  defaultSubnetAccessMode: Private
+  defaultSubnetSize: 32
+  podSubnetAccessMode: Private
+  vpcConnectivityProfile: /orgs/default/projects/wenqi-test/vpc-connectivity-profiles/default
```

Comment:
pre-created VPC does not have vpcConnectivityProfile, right?

Reply:
No, it is not set in pre-created VPC case.